spec/migrations/20191118161319_subclass_resource_pools_spec.rb

```diff
@@ -0,0 +1,42 @@
+require_migration
+
+describe SubclassResourcePools do
+  let(:ext_management_system_stub) { migration_stub(:ExtManagementSystem) }
+  let(:resource_pool_stub)         { migration_stub(:ResourcePool) }
+
+  migration_context :up do
+    it "migrates resource pools from all supported providers" do
```

Comment:
It would probably also be good to have a test for a record that isn't changed by the migration.

Reply:
:+1: done